source/connection/flow/i_flow.hpp

```diff
@@ -19,6 +19,8 @@ class IFlow : public Identifiable {
     virtual SizeByte get_sending_quota() const = 0;
     virtual std::optional<TimeNs> get_last_rtt() const = 0;
     virtual SizeByte get_delivered_data_size() const = 0;
+    virtual SizeByte get_sent_data_size() const = 0;
+    virtual uint32_t retransmit_count() const = 0;
```

Comment:
I think total size of retransmitted data is more representative
```suggestion
    virtual SizeByte retransmit_count() const = 0;
```